README.md

```diff
@@ -2,55 +2,48 @@
-framework, PROOF, that can considerably speed up an analysis.
+framework, RDataFrame, that can considerably speed up an analysis, taking 
+full advantage of multi-core and distributed systems.
 
-Thanks to the built-in C++ interpreter cling, the command, the
-scripting and the programming language are all C++. The interpreter
-allows for fast prototyping of the macros since it removes the time
-consuming compile/link cycle. It also provides a good environment to
-learn C++. If more performance is needed the interactively developed
-macros can be compiled using a C++ compiler via a machine independent
-transparent compiler interface called ACliC.
+ROOT is written in C++, and, thanks to PyROOT, provides a powerful Python interface 
+thanks to which it's possible to use all of its components seamlessly from Python.
+This is possible thanks to the powerful built-in C++ interpreter, Cling, that allows, 
+among the other things, to create dynamic python bindings at runtime, a C++ interactive 
+prompt as well as rapid C++ prototyping.
 
-The system has been designed in such a way that it can query its databases
-in parallel on clusters of workstations or many-core machines. ROOT is
-an open system that can be dynamically extended by linking external
-libraries. This makes ROOT a premier platform on which to build data
-acquisition, simulation and data analysis systems.
+ROOT fully embraces the open-source philosophy, it's made with passion by its community 
+for the benefit of its community.
 
 [![License: LGPL v2.1+](https://img.shields.io/badge/License-LGPL%20v2.1+-blue.svg)](https://www.gnu.org/licenses/lgpl.html)
 [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/5060/badge)](https://bestpractices.coreinfrastructure.org/projects/5060)
 
-## Cite
+## Contribution Guidelines
+- [How to contribute](https://github.com/root-project/root/blob/master/CONTRIBUTING.md)
+- [Coding conventions](https://root.cern/coding-conventions)
+- [Meetings](https://root.cern/meetings)
 
+## Cite
 When citing ROOT, please use both the reference reported below and the DOI specific to your ROOT version available [on Zenodo](https://zenodo.org/badge/latestdoi/10994345) [![DOI](https://zenodo.org/badge/10994345.svg)](https://zenodo.org/badge/latestdoi/10994345). For example, you can copy-paste and fill in the following citation:
 
     Rene Brun and Fons Rademakers, ROOT - An Object Oriented Data Analysis Framework,
     Proceedings AIHENP'96 Workshop, Lausanne, Sep. 1996,
     Nucl. Inst. & Meth. in Phys. Res. A 389 (1997) 81-86.
-    See also "ROOT" [software], Release vX.YY/ZZ, dd/mm/yyyy,
-    (Select the right link for your release here: https://zenodo.org/search?page=1&size=20&q=conceptrecid:848818&all_versions&sort=-version).
-
-## Live Demo for CERN Users
-[![](https://swanserver.web.cern.ch/swanserver/images/badge_swan_white_150.png)](http://cern.ch/swanserver/cgi-bin/go?projurl=https://github.com/cernphsft/rootbinder.git)
```

Comment:
The Swan demo seems to still work (the binder do seem to no longer work i.e. need some update).  What is the benefit of removing the link?

Reply:
That seemed to be too CERN-centric. I might have been overzealous though: do you think it can be re-added?